src/Microsoft.PowerShell.Commands.Utility/commands/utility/convert-HTML.cs

```diff
@@ -341,10 +341,20 @@ private void InitializeResolvedNameMshParameters()
                 foreach (PSPropertyExpression resolvedName in resolvedNames)
                 {
                     Hashtable ht = CreateAuxPropertyHT(label, alignment, width);
-                    ht.Add(FormatParameterDefinitionKeys.ExpressionEntryKey, resolvedName.ToString());
+                    if (resolvedName.Script != null)
+                    {
+                        // script block
+                        ht.Add(FormatParameterDefinitionKeys.ExpressionEntryKey, resolvedName.Script);
+                    }
+                    else
+                    {
+                        // property name
```

Comment:
I'd remove the obvious comment and enhance comment in line 346.

Reply:
@iSazonov: Done. Is it sufficient to add "[Feature]" in the first line of a commit message _once_?